# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1485557530 18000
#      Fri Jan 27 17:52:10 2017 -0500
# Node ID 48a0bb5d51503adb0fe771b78a46d7bd96adf317
# Parent  3f72b4b9b50f74554e6ac9cdcaea592b58305478
don't add __shutdown_qt__ to list of atexit functions inside __shutdown_qt__

* __init_qt__.cc (__shutdown_qt__): Don't add __shutdown_qt__ to list
of atexit functions.

diff --git a/libgui/graphics/__init_qt__.cc b/libgui/graphics/__init_qt__.cc
--- a/libgui/graphics/__init_qt__.cc
+++ b/libgui/graphics/__init_qt__.cc
@@ -106,18 +106,16 @@ namespace QtHandles
 
   bool
   __shutdown__ (void)
   {
     if (qtHandlesInitialized)
       {
         gh_manager::auto_lock lock;
 
-        octave_add_atexit_function ("__shutdown_qt__");
-
         gtk_manager::unload_toolkit ("qt");
 
         gh_manager::enable_event_processing (false);
 
         qtHandlesInitialized = false;
 
         return true;
       }
